backend/src/v5/models/projects.js

```diff
@@ -36,6 +36,13 @@ Projects.removeModelFromProject = (ts, project, model) => updateOneProject(
 	{ $pull: { models: model } },
 );
 
+Projects.checkProjectExists = async (ts, project, projection) => {
+	const res = await findOneProject(ts, { _id: project }, projection);
```

Comment:
is this not identical to getProjectById just below it? I think we can just use that instead?